### PR TITLE
8340974: Ambiguous name of jtreg property vm.libgraal.enabled

### PR DESCRIPTION
**Pull request by toddjonker**
This disambiguates the situation where libgraal is "enabled" for use by non-CompilerBroker compilations, without being used as the JIT compiler.

Per discussion at https://github.com/openjdk/jdk/pull/21120#issuecomment-2372462365 

Grep shows that the property and related method are not used in this codebase.  Tier1 tests on linux-x86_64-server-release pass cleanly.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340974](https://bugs.openjdk.org/browse/JDK-8340974): Ambiguous name of jtreg property vm.libgraal.enabled (**Enhancement** - P4)

### Reviewers
 * [Doug Simon](https://openjdk.org/census#dnsimon) (@dougxc - **Reviewer**)
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21190/head:pull/21190` \
`$ git checkout pull/21190`

Update a local copy of the PR: \
`$ git checkout pull/21190` \
`$ git pull https://git.openjdk.org/jdk.git pull/21190/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21190`

View PR using the GUI difftool: \
`$ git pr show -t 21190`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21190.diff">https://git.openjdk.org/jdk/pull/21190.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21190#issuecomment-2375314272)

### CODE REVIEW

File: test/jtreg-ext/requires/VMProps.java [hunk 1]
```diff
@@ -560,7 +559,7 @@ protected String hasLibgraal() {
      *
      * @return true if libgraal is used as JIT compiler.
      */
-    protected String isLibgraalEnabled() {
+    protected String isLibgraalJit() {
```
Comment (dougxc) [inline test/jtreg-ext/requires/VMProps.java:562]: I slightly prefer `isLibgraalJIT` as this acronym is (most) capitalized in the code base.

You should also rename `isLibgraalEnabled` to `isLibgraalJIT` in `test/lib/jdk/test/whitebox/code/Compiler.java` for consistency.

Comment (toddjonker) [reply test/jtreg-ext/requires/VMProps.java]: Renamed both. I was being conservative by limiting scope of change to the relevant interface, and felt that the existing name was reasonable in the `Compiler` scope.

**Review by dougxc — APPROVED** [review]
LGTM.

**Review by phohensee — APPROVED** [review]

**Pull request closed without merging**